srcpkgs/apt/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'apt'
 pkgname=apt
-version=2.0.0
+version=2.1.6
 revision=1
 build_style=cmake
 configure_args="-DDPKG_DATADIR=/usr/share/dpkg -DCURRENT_VENDOR=debian
```

Comment:
Is it correct to claim `-DCURRENT_VENDOR=debian`?

Reply:
Maybe it is better to remove it. It does not exist in [CMakeLists.txt](https://github.com/Debian/apt/blob/master/CMakeLists.txt)...

Reply:
I removed both -DCURRENT_VENDOR=debian and -DDOCBOOK_XSL=/usr/share/xsl/docbook . They do nothing, vendor is autodetected as debian in the configure stage.

Reply:
They are needed when cross-compiling...